src/asmcnc/apps/systemTools_app/screens/screen_build_info.py

```diff
@@ -455,6 +457,20 @@
                         on_press: root.open_data_consent_app()
                         color: hex('#f9f9f9ff')
                         markup: True
+                        
+                    ToggleButton:
+                        id: toggle_ssh_button
+                        size_hint: (None,None)
+                        height: dp(35)
+                        width: dp(180)
+                        background_normal: "./asmcnc/apps/systemTools_app/img/word_button.png"
+                        background_down: "./asmcnc/apps/systemTools_app/img/word_button.png"
+                        border: [dp(7.5)]*4
+                        center: self.parent.center
+                        pos: self.parent.pos
+                        on_press: root.toggle_ssh()
+                        color: hex('#f9f9f9ff')
+                        markup: True
```

Comment:
Have you tested that there's still plenty of space for the textbox under the "Advanced"" button? Probs needs a few lines - need to be able to see the branch and the commit code

Reply:
Yep, text fits fine.